datalad/core/local/save.py

```diff
@@ -196,7 +196,14 @@ def __call__(path=None, message=None, dataset=None,
                 untracked=untracked_mode,
                 recursive=recursive,
                 recursion_limit=recursion_limit,
+                on_failure='ignore',
                 result_renderer='disabled'):
+            if s['status'] == 'error':
+                # Downstream code can't do anything with these. Let the caller
+                # decide their fate.
+                yield dict(s, action='save')
```

Comment:
Is it a typical workflow with the return dictionaries to just replace their `action` field? IMHO might contribute to more rabbit holes. Eg what if status would start yielding from yet another command? 
My point is - why not to just reyield an original record and then may be an additional status error if so desired?

Reply:
> why not to just reyield an original record 

I went back and forth on it, but decided to yield with the `save` action type because nowhere else does `save` yield `status` results directly and it looked odd to me as the caller.  And its the action of saving that the caller ultimately cares about.

I'm unconvinced by your rabbit-hole motivation, but I don't particularly care one way or the other.  I'll update it to yield the unchanged record.
 
> and then may be an additional status error if so desired?

Please, no.  We're already hitting the user with two messages about the same error.

Reply:
> I'm unconvinced by your rabbit-hole motivation, but I don't particularly care one way or the other. I'll update it to yield the unchanged record.

my neither, and you put more thought into it than I did, so please proceed the way you think is the best.  I just was surprised that we do such "impersonations" ;)

Reply:
I even start to wonder if we should add additional field to the status records, like `"chain"` in run (or `"original"`), to possibly group those original messages, so it would become clear which other actions were result of a given action or caused an error (and by default not render them at all).

Reply:
> my neither, and you put more thought into it than I did, so please proceed the way you think is the best.

It's already been updated, and, as I said, I don't feel strongly one way or the other, so I'll plan to leave it be.